doc/features/feature-api.md

```diff
@@ -51,6 +51,20 @@ To force a new request set `force_newcert` to `true`:
 force_newcert: true
 ```
 
+To increase your security set `ca_fingerprint` to validate the certificate of the `ca_host`:
+
+```
+ca_fingerprint: "00 DE AD BE EF"
+# alternatively
+ca_fingerprint: "00:DE:AD:BE:EF"
```

Comment:
finger print can also be lowercase, cause it will be made uppercase by the filter. 

`ca_fingerprint: 00 de ad be ef`